src/test-helpers.js

```diff
@@ -72,8 +72,53 @@ function findNcListItems(wrapper, text) {
 		: listItems.filter(listItem => listItem.vm.title === text)
 }
 
+/**
+ *
+ * @param {object} data response from the server
+ * @param {object} [data.headers = {}] headers of response
+ * @param {object} [data.payload = {}] payload of response
+ * @param {number} [data.status = 200] status code of response
+ * @return {object}
+ */
+function generateOCSResponse({ headers = {}, payload, status = 200 }) {
+	return {
+		headers,
+		status,
+		data: {
+			ocs: {
+				data: Array.isArray(payload) ? [...payload] : { ...payload },
+			},
+		},
+	}
+}
+
+/**
+ *
+ * @param {object} data response from the server
+ * @param {object} [data.headers = {}] headers of response
+ * @param {object} [data.payload = {}] payload of response
+ * @param {number} data.status status code of response
+ * @return {object}
+ */
+function generateOCSErrorResponse({ headers = {}, payload, status }) {
+	return {
+		headers,
+		status,
+		response: {
+			status,
+			data: {
+				ocs: {
+					data: Array.isArray(payload) ? [...payload] : { ...payload },
+				},
+			},
+		},
+	}
+}
```

Comment:
I faced a couple of times a situation, when server response was mutated, and test passed because of that, although it didn't.
It probably takes more resources to compare output with expected, than by reference, but i think, that it's better to have the mocked data immutable

Changed to `cloneDeep` until further discussion

Reply:
> I faced a couple of times a situation, when server response was mutated, and test passed because of that, although it didn't.

I'd say that this is an issue of the test. For example, using one object for response twice and probably making several actual tests inside one test.